app/src/components/common/AppHeader.vue

```diff
@@ -88,6 +91,13 @@ export default {
     isDevelopmentMode() {
       return this.session.developmentMode
     },
+    hideSidebarMenu() {
+      return (
+        (this.$route.name === `paywall` ||
```

Comment:
how about adding a property to routes that shouldn't show the signbar instead of listing the routes here? this always requires a double maintenance

Reply:
great suggestion. thanks.